File: 2019/1/1.2.cpp
#include "input.h"
#include <iostream>

int main() {
  int sum = 0;
  for (auto x : input) {
    auto y = (x/3)-2;
    do { sum += y; y = (y/3)-2; } while (y>0);
  }
  std::cout << sum << std::endl;
  return 0;
}

File: 2022/1/1.2.py
max = []
sum = 0
with open("input.txt",'r') as f:
  for line in f:
    line = line.strip()
    if line == '':
      max.append(sum)
      sum = 0
    else:
      sum += int(line)
max = sorted(max,reverse=True)
print(max[0]+max[1]+max[2])
